storage/innobase/handler/ha_innodb.cc

```diff
@@ -5684,31 +5644,15 @@ static int innodb_init(void *p) {
   innobase_hton->unlock_hton_log = innobase_unlock_hton_log;
   innobase_hton->collect_hton_log_info = innobase_collect_hton_log_info;
   innobase_hton->fill_is_table = innobase_fill_i_s_table;
-<<<<<<< HEAD
-  innobase_hton->flags = HTON_SUPPORTS_EXTENDED_KEYS |
-                         HTON_SUPPORTS_FOREIGN_KEYS | HTON_SUPPORTS_ATOMIC_DDL |
-                         HTON_CAN_RECREATE | HTON_SUPPORTS_SECONDARY_ENGINE |
-                         HTON_SUPPORTS_TABLE_ENCRYPTION |
-                         HTON_SUPPORTS_ONLINE_BACKUPS | HTON_SUPPORTS_COMPRESSED_COLUMNS |
-                         HTON_SUPPORTS_GENERATED_INVISIBLE_PK;
 #ifdef WITH_WSREP
   innobase_hton->flags |= HTON_WSREP_REPLICATION;
 #endif /* WITH_WSREP */
-||||||| merged common ancestors
-  innobase_hton->flags = HTON_SUPPORTS_EXTENDED_KEYS |
-                         HTON_SUPPORTS_FOREIGN_KEYS | HTON_SUPPORTS_ATOMIC_DDL |
-                         HTON_CAN_RECREATE | HTON_SUPPORTS_SECONDARY_ENGINE |
-                         HTON_SUPPORTS_TABLE_ENCRYPTION |
-                         HTON_SUPPORTS_ONLINE_BACKUPS | HTON_SUPPORTS_COMPRESSED_COLUMNS |
-                         HTON_SUPPORTS_GENERATED_INVISIBLE_PK;
-=======
   innobase_hton->flags =
```

Comment:
This will overwrite the flag set above in ifdef WITH_WSREP.
WITH_WSREP block should go after this assignment.